account_financial_report_qweb/__openerp__.py

```diff
@@ -35,8 +35,11 @@
         'report/templates/trial_balance.xml',
         'view/account_view.xml'
     ],
-    'test': [
-    ],
+    'external_dependencies': {
+        "python": [
+            "xlsxwriter",
+        ],
+    },
```

Comment:
AFAICS xslxwriter is not a direct dependency of afr_qweb. It is a dependency of report_xslx, so it should not be declared here.